.circleci/config.yml

```
@@ -111,3 +111,56 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
+      - architect/push-to-app-collection:
+          context: architect
+          name: push-oauth2-proxy-to-capz-app-collection
+          app_name: "oauth2-proxy"
+          app_namespace: "monitoring"
+          app_collection_repo: "capz-app-collection"
+          requires:
+            - push-to-control-plane-app-catalog
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*/
```

I know, the rest of this file doesn't look any better, but maybe you can give some love to it. Otherwise still LGTM. :)

```suggestion
      - architect/push-to-app-collection:
          name: push-to-capz-app-collection
          context: architect
          app_name: oauth2-proxy
          app_namespace: monitoring
          app_collection_repo: capz-app-collection
          requires:
            - push-to-control-plane-app-catalog
          filters:
            branches:
              ignore: /.*/
            tags:
              only: /^v.*/
```